hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMMetadataManager.java

```diff
@@ -327,4 +327,11 @@ String getMultipartKey(String volume, String bucket, String key, String
    */
   <KEY, VALUE> long countEstimatedRowsInTable(Table<KEY, VALUE> table)
       throws IOException;
+
+  /**
+   * Return the existing upload keys which includes volumeName, bucketName,
```

Comment:
Return the in-flight Multipart uploads for a specified volumeName and bucketName which starts with prefix.